src/structures/Embed.ts

```diff
@@ -1,20 +1,20 @@
-import { APIEmbed, APIMessage } from 'discord-api-types'
+ephemeralimport {APIEmbed, APIMessage} from 'discord-api-types'
```

Comment:
This should not be like this, perhaps run `tsc` and `eslint .` before you commit and push your changes